### PR TITLE
AP_UAVCAN: add BatteryInfoAux dsdl message

### PR DESCRIPTION
**Pull request by bugobliterator**
This message includes some information that is not present in https://github.com/UAVCAN/public_regulated_data_types/blob/legacy-v0/uavcan/equipment/power/1092.BatteryInfo.uavcan

### CODE REVIEW

File: libraries/AP_UAVCAN/dsdl/ardupilot/equipment/power/20004.BatteryInfoAux.uavcan [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Please ensure that this message is sent before the BatteryInfo which doesn't
+# include timestamp, following field should contain timestamp of last current measurement
+uavcan.Timestamp timestamp
+
+# Battery individual cell voltages
+# length of following field also used as cell count
+float16[<=255] voltage_cell          # [Volt]
+
+# Number of Charge Discharge Cycle
+# A charge cycle should happen when system uses all of the battery’s power.
```
Comment (mxiaogit) [inline libraries/AP_UAVCAN/dsdl/ardupilot/equipment/power/20004.BatteryInfoAux.uavcan:15]: There is a problem with the text encoding in the comment, please change "‘" to "'".

Comment (mxiaogit) [inline libraries/AP_UAVCAN/dsdl/ardupilot/equipment/power/20004.BatteryInfoAux.uavcan:15]: Text encoding comments have questions, please "’" is modified to "'".

**Review by tridge — APPROVED** [review]

**Comment by peterbarker** [comment]
@bugobliterator same reasonable changes requested here....

**Comment by peterbarker** [comment]
Merged, thanks!

Follow-up PR to fix those comments would be good at some stage :-)

**Pull request merged**